test/e2e/catalog_e2e_test.go

```diff
@@ -7,22 +7,26 @@ import (
 	"fmt"
 	"net"
 	"os"
+	"strconv"
 	"strings"
 	"time"
 
 	"github.com/blang/semver/v4"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	"github.com/operator-framework/api/pkg/lib/version"
```

Comment:
nit: import order (looks like it was corrected in a few other places)

Reply:
yes it was will fix

Reply:
fixed